README.md

```diff
@@ -25,21 +25,22 @@ Chartpress can do the following with the help of some configuration.
 Chartpress will infer chart versions and image tags using a few key pieces of
 information.
 
-1. `tag`: The latest commit that is tagged on the current branch, or 0.0.1 if no
-   tag was found.
-2. The latest commit that influenced anything on a path within the git
-   repository that matters to the chart version or image tag. The paths that
-   matters is determined using the image build contexts and additional specified
-   paths.
-   1. `n`: The latest commits commit distance count since the tag, described as
-      3 or more numbers, prefixed with n.
-   2. `h`: The latest commits abbreviated SHA hash, which is typically 7
-      characters, prefixed with h.
-3. If `--long` is specified or not. When it is specified tagged commits will be
-   written out with `n000.h<hash>` appended to it.
-4. If `tag` contains a `-`, `tag.n.h` will be used, and if not, `tag-n.h` will
-   be used. There should be exactly one `-` in the final version specification
-   to become a valid SemVer2 version.
+1. `tag`: If not directly set by `--tag`, it will be inferred from most recent
+   commit that is tagged in the _current branch_, or be set to 0.0.1 if no
+   commit is tagged.
+   1. If the `--tag` flag starts with a `v`, it will be removed when setting the
+      Chart.yaml version specifically. An error will be raised if it still isn't
+      a valid SemVer2 version because Helm 3 requires it.
```

Comment:
```suggestion
   1. If the `tag` has a leading `v` but is otherwise a valid
      [SemVer2](https://semver.org) version, it will be stripped from Chart.yaml
      before its set as Helm 3 requires Helm chart versions to be SemVer2
      compliant.
```